src/routes/news/components/NewsElement.js

```diff
@@ -0,0 +1,105 @@
+// @flow
+
+import * as React from 'react'
+import styled from 'styled-components'
+import { LocalNewsModel } from '@integreat-app/integreat-api-client'
+import CleanLink from '../../../modules/common/components/CleanLink'
+import LastUpdateInfo from './../../../modules/common/components/LastUpdateInfo'
+import type { TFunction } from 'react-i18next'
+import { textTruncator } from './../../../modules/theme/constants/helpers'
+
+const TRUNCATE_LETTERS_COUNT = 30
+
+const Link = styled(CleanLink)`
+  display: flex;
+  background-color: white;
+`
+
+const ReadMoreLink = styled(CleanLink)`
+  align-self: flex-end;
+  color: ${({ theme }) => (theme.colors.themeColor)};
+  font-weight: 600;
+`
+
+const Description = styled.div`
+  display: flex;
+  height: 100%;
+  min-width: 1px; /* needed to enable line breaks for too long words, exact value doesn't matter */
+  flex-direction: column;
+  flex-grow: 1;
+  padding: 15px 10px 0;
+  word-wrap: break-word;
+
+  > * {
+    padding-bottom: 10px;
+  }
+`
+
+const Title = styled.h3`
+  margin-bottom: 0;
+  color: ${({ theme }) => (theme.colors.headlineTextColor)};
+  font-family: Raleway;
+  font-size: 18px;
+  font-weight: 700;
+`
+const Body = styled.p`
+  color: ${({ theme }) => (theme.colors.headlineTextColor)};
+  font-size: 16px;
+  line-height: 1.38;
+`
+
+const StyledNewsElement = styled.div`
+  padding-bottom: 2px;
+  background: linear-gradient(to left, rgba(168, 168, 168, 0.2), #bebebe 51%, rgba(168, 168, 168, 0.2));
```

Comment:
Where do you get these colors from?

Reply:
These are the colors for the `linear gradient` for the horizontal line separator between the list items (from the designs)

Reply:
We've uploaded the designs to  `Zeplin` . And that's how I got these colors.
I can check the current theme color and pick something close to the design.

[
![Annotation 2020-05-14 124042](https://user-images.githubusercontent.com/28042978/81919207-343f5480-95e0-11ea-8916-98b0c2719f84.jpg)
](url)

Reply:
Okay, sorry for our bad design and communication here then, I would appreciate to use a similar color from our theme. (with exception of the two colors for TüNews).

Reply:
Just to double check. you mean by `two colors for TüNews` the light and dark blueish colors.
 
But the  `content` and `title` color will be `black` as the rest of the website. right?

Reply:
@klinzo also should I keep the color gradient for the items horizontal separator or replace it with a solid border with a color from the theme maybe `textDisabledColor`

Reply:
Didn't know about the light blue. So if you need the grey tünews color too, you can also add it to the theme. If you'd say it looks good enough to use one of our defined colors, use those.

Reply:
Is there a reason why we should not keep it? Imo it is fine :+1:

Reply:
@klinzo I think we should keep it as well. Anything still needs to be updated for this comment?